src/framework/app-base.js

```diff
@@ -1894,7 +1894,12 @@ class AppBase extends EventHandler {
      * @param {Layer} [layer] - The layer to render the texture into. Defaults to {@link LAYERID_IMMEDIATE}.
      * @ignore
      */
-    drawTexture(x, y, width, height, texture, material, layer = this.scene.defaultDrawLayer) {
+    drawTexture(x, y, width, height, texture, material, layer = this.scene.defaultDrawLayer, filterable = true) {
```

Comment:
Shouldn't filterable parameter be documented? It'll show up in Intellisense right?

Reply:
good point, done